README.md

```diff
@@ -20,6 +21,14 @@ For the driver, change the library name in privateExportsConfigs, the driver nam
 For the maven artifact names, those are all in publish.gradle about 40 lines down.
 
 ## Building and editing
+
 This uses gradle, and uses the same base setup as a standard GradleRIO robot project. This means you build with `./gradlew build`, and can install the native toolchain with `./gradlew installRoboRIOToolchain`. If you open this project in VS Code with the wpilib extension installed, you will get intellisense set up for both C++ and Java.
 
-By default, this template builds against the latest WPILib development build. To build against the last WPILib tagged release, build with `./gradlew build -PreleaseMode`.
\ No newline at end of file
+By default, this template builds against the latest WPILib development build. To build against the last WPILib tagged release, build with `./gradlew build -PreleaseMode`.
+
+### How to fix broken CI for post-season development builds
+
+There's two ways to fix builds after WPILib releases an alpha version for the next year and deletes the previous year's development artifacts.
+
+1. The short-term fix to keep building with the previous year is modifying build.gradle to use `wpilibRepositories.addAllReleaseRepositories(project)` or adding `-PreleaseMode` to CI build invocations
```

Comment:
This is no longer an issue.  All releases are also visible in the development repo.